Source/bucketalloc.c

```diff
@@ -55,14 +55,14 @@ struct BucketAlloc
 
 static int CreateBucket( struct BucketAlloc* ba )
 {
-	size_t size;
+	unsigned int size;
```

Comment:
Can you talk me through your decision here and other alternatives you considered?